Query/objects/domain_intelligence.json

```diff
@@ -0,0 +1,40 @@
+{
+	"description": "Insights from threat intelligence platforms about domains",
+	"caption": "Domain Threat Intelligence",
+	"name": "domain_intelligence",
+	"attributes": {
+		"domain": {
+			"requirement": "optional",
+		},
+		"domain_info": {
+			"requirement": "optional",
+		},
+		"answers": {
+			"requirement": "optional",
+		},
```

Comment:
VirusTotal provides similar data listing (some of?) the DNS records for a domain. You can see some in the examples of Mahesh's spec: https://docs.google.com/document/d/1WkDO9Vnc5m3JZivipKVu7MoGpxyJrr19nZtkb-A-MxU/edit#heading=h.52d8ut26v49d

Reply:
Since the description of this attribute inherited from dictionary.json is "DNS Answer: The Domain Name System (DNS) answers.", it's not really clear what the purpose is, since this is no corresponding DNS request.... so, what is it answering?

Looking at Mahesh's document, it appears that VirusTotal is not providing passive DNS logs; instead, it's periodically caching the DNS entries for a domain. I think a better place to represent this is on the `DomainInfo` object.

Reply:
It's answers to past DNS requests. The data VT provides here, as well as the passive DNS logs from AlienVault, fits the DNS Answer object but does not fit Domain Information. Are you suggesting it be named something else?

Reply:
Sorry I wasn't clear. I think the data from VirusTotal is bettered modeled at DomainInfo.dns_entries or something, with the type of DnsAnswer. That would allow us to add live DNS lookups for any domain, regardless of the provider.

For VirusTotal, the data would come from VirusTotal -- maybe it's not real-time, but we know the last time it was updated and can report that in raw_data. In the other cases, we could build our own DNS driver that supports only `DnsAnswer`, and then use federated joins to enable lookups on `DomainInfo` objects from any platform.

Those would be semantically different from passive DNS logs though. For passive DNS logs, it makes more sense to represent that as part of the threat intel model. Those are an awkward fit anywhere though, because they are sort of a DnsActivity event, but we probably cannot query them like events

Reply:
On the threat intel object, if we want to represent passive DNS information, then I would suggest renaming this to `passive_dns_logs` or similar.